src/App.js

```diff
@@ -1,12 +1,26 @@
 /* eslint-disable react/prefer-stateless-function */
 
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
 class App extends React.Component {
   render() {
     return (
-      <Calculator />
+      // <Calculator />
+      <Router>
+        <Routes>
+          <Route path="Quotes" element={<Quotes />} />
+          <Route path="Calculator" element={<Calculator />} />
+          <Route path="Home" element={<Home />} />
+        </Routes>
+      </Router>
```

Comment:
- [ ] When I start the app it shows me an empty page. 

![Capture](https://user-images.githubusercontent.com/64173189/159545012-cc659d81-c75c-4d73-b471-499bd44a67e7.JPG)

That is because when the app starts point to `http://localhost:3000/` and you don't have content for this route.

It would be better if when someone starts the app it shows content. Kindly fix this minor issue by just adding one route pointing to  `/ ` 

```suggestion
        <Routes>
          <Route path="/" element={<Quotes />} />
          <Route path="Calculator" element={<Calculator />} />
          <Route path="Home" element={<Home />} />
        </Routes>
      </Router>
```

Don't forget to change the path of your navbar. 😄 👍